js/api/Triggers.js

```diff
@@ -32,13 +32,54 @@ export function loadTriggers(form, survey, callback) {
   })
 }
 
-// Fetches the cached forms related to a specific survey
-export function loadCachedTrigger(formId) {
-  return realm.objects('TimeTrigger').filtered(`formId= "${formId}"`)
+// Fetches the cached triggers related to a specific survey.
+export function loadCachedTriggers(formId) {
+  return realm.objects('TimeTrigger').filtered(`formId= "${formId}"`);
 }
 
 /**
- * Fetches all cached geofence triggers
+ * Fetches all cached datetime triggers for the accepted surveys
+ * @return {object}  Realm object containing an array of 'TimeTrigger' objects,
+ */
+export function loadCachedTimeTriggers(options = {}, callback) {
+  loadAllAcceptedSurveys((err, response) => {
+    if (err) {
+      console.warn(err);
+      callback(err, []);
+      return
+    }
+    
+    try {
```

Comment:
Any particular reason we are surrounding all this code in a try/catch?  I do not see any methods that throw an error.